docs/source/cli/model-training-inference/configuration-run.rst

```diff
@@ -397,14 +397,14 @@ General Configurations
             - For link prediction tasks, the default value is ``mrr``.
 - **gamma**: Set the value of the hyperparameter denoted by the symbol gamma. Gamma is used in the following cases: i/ focal loss for binary classification ii/ DistMult score function for link prediction, iii/ TransE score function for link prediction, iv/ RotatE score function for link prediction, v/ shrinkage loss for regression.
 
-    - Yaml: ``gamma: 10.0``
-    - Argument: ``--gamma 10.0``
-    - Default value: None
+    - Yaml: ``gamma: 2.0``
+    - Argument: ``--gamma 2.0``
+    - Default value: ``2``
```

Comment:
The default is None. For different loss func, the default value is different.

Reply:
I would suggest we give a default value to `gamma`, otherwise users cannot see the effect of `focal loss` or `shrinkage loss`.

Reply:
revised default values according to different usages.